README.md

```diff
@@ -1,70 +1,33 @@
-# LiveCompositor
+<h1 align="center">
+  <img src="assets/lc_logo_large.svg" width=600 alt="LiveCompositor">
 
-![LiveCompositor logo](./assets/lc_logo_large.svg)
+  <a href="https://compositor.live/docs">Documentation</a> |
+  <a href="https://discord.gg/Cxj3rzTTag">Discord</a>
+</h1>
 
-LiveCompositor is an open-source media server for real-time, low-latency, programmable video and audio mixing/composing.
+LiveCompositor is an open-source media server for real-time, low-latency, programmable video and audio mixing.
 
-LiveCompositor targets real-time use cases, like video conferencing, live-streaming, or broadcasting (e.g. with [WebRTC](https://en.wikipedia.org/wiki/WebRTC) / [HLS](https://en.wikipedia.org/wiki/HTTP_Live_Streaming) / [RTMP](https://en.wikipedia.org/wiki/Real-Time_Messaging_Protocol)), however [offline processing](https://compositor.live/docs/deployment/configuration#live_compositor_offline_processing_enable) is also available.
+LiveCompositor targets real-time use cases, with a significant focus on situations where latency is critical. It is a great fit
+for any video conferencing, live-streaming, or broadcasting solutions where you need to combine or modify video on the fly.
```

Comment:
To avoid double `or`:
```suggestion
for any video conferencing, live-streaming, or broadcasting solutions where you need to combine and modify video on the fly.
```
or
```suggestion
for any video conferencing, live-streaming, and broadcasting solutions where you need to combine or modify video on the fly.
```

Reply:
I think it is fine in this case